# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/x11/xfree86/sparc-keyboard-def-26headers.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- xc/programs/Xserver/hw/xfree86/os-support/linux/lnx_kbd.c~	2004-08-21 16:22:07.000000000 +0200
+++ xc/programs/Xserver/hw/xfree86/os-support/linux/lnx_kbd.c	2004-08-21 19:54:02.000000000 +0200
@@ -102,7 +102,7 @@
 
 /* Deal with spurious kernel header change */
 #if defined(LINUX_VERSION_CODE) && defined(KERNEL_VERSION)
-# if LINUX_VERSION_CODE >= KERNEL_VERSION(2,5,42)
+# if LINUX_VERSION_CODE >= KERNEL_VERSION(2,5,42) && !defined(__sparc__)
 #  define rate period
 # endif
 #endif
--- xc/programs/Xserver/hw/xfree86/os-support/linux/lnx_io.c~	2004-08-21 16:22:07.000000000 +0200
+++ xc/programs/Xserver/hw/xfree86/os-support/linux/lnx_io.c	2004-08-21 19:53:22.000000000 +0200
@@ -75,7 +75,7 @@
 
 /* Deal with spurious kernel header change */
 #if defined(LINUX_VERSION_CODE) && defined(KERNEL_VERSION)
-# if LINUX_VERSION_CODE >= KERNEL_VERSION(2,5,42)
+# if LINUX_VERSION_CODE >= KERNEL_VERSION(2,5,42) && !defined(__sparc__)
 #  define rate period
 # endif
 #endif
